src/index.ts

```diff
@@ -254,7 +254,7 @@ prog
         peerDependencies: template === 'react' ? { react: '>=16' } : {},
         husky: {
           hooks: {
-            'pre-commit': 'pretty-quick --staged',
```

Comment:
This fixes a bug in the implementation of #99 that I mentioned in #7. #99 added eslint, but husky was incorrectly only running prettier instead of eslint+prettier. By calling the `tsdx lint` command instead, it's doing the full linting.